client/my-sites/customer-home/cards/tasks/go-mobile/index.js

```diff
@@ -53,4 +52,9 @@ const GoMobile = () => {
 		/>
 	);
 };
+
+GoMobile.propTypes = {
+	isIos: PropTypes.bool,
```

Comment:
Non-blocking and optional: I'd add another prop for Android. It's a bit more of boilerplate, but I think the code ends up being more readable if we are explicit.

```
if ( ! isIos && | ! isAndroid ) {
  return null;
}

if ( isIos ) {
    return <AppsBadge storeName={ 'ios' } ... >
}

if ( isAndroid ) {
    return <AppsBadge storeName={ 'android' } ... >
}
```